tests/unit/routes/app/templates/account/billing-info/invoices/list-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | app/templates/account/billing-info/invoices/list', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    let route = this.owner.lookup('route:app/templates/account/billing-info/invoices/list');
```

Comment:
It would be great if you can you write render test and Table test as done in others components rather than route test .